tests/js/spec/views/organizationGroupDetails/groupEventDetailsContainer.spec.tsx

```diff
@@ -0,0 +1,103 @@
+import {render, screen} from 'sentry-test/reactTestingLibrary';
+
+import OrganizationEnvironmentsStore from 'sentry/stores/organizationEnvironmentsStore';
+import GroupEventDetailsContainer, {
+  GroupEventDetailsProps,
+} from 'sentry/views/organizationGroupDetails/groupEventDetails';
+import {ReprocessingStatus} from 'sentry/views/organizationGroupDetails/utils';
+
+jest.mock(
+  'sentry/views/organizationGroupDetails/groupEventDetails/groupEventDetails',
+  () => () => <div>GroupEventDetails</div>
+);
+
+const makeProps = (props: Partial<GroupEventDetailsProps>): GroupEventDetailsProps => {
+  const mergedProps: GroupEventDetailsProps = {
+    event: TestStubs.Event(),
+    eventError: false,
+    group: TestStubs.Group(),
+    loadingEvent: false,
+    onRetry: () => null,
+    groupReprocessingStatus: ReprocessingStatus.NO_STATUS,
+    organization: TestStubs.Organization(),
+    project: TestStubs.Project(),
+    params: {groupId: '0', orgId: '0', eventId: '0'},
+    ...props,
+  } as GroupEventDetailsProps;
+
+  return mergedProps;
+};
+
+describe('groupEventDetailsContainer', () => {
+  const organization = TestStubs.Organization();
+
+  beforeEach(() => {
+    OrganizationEnvironmentsStore.init();
+  });
+
+  afterEach(() => {
+    OrganizationEnvironmentsStore.teardown();
+  });
+
+  it('fetches environments', async function () {
+    const environmentsCall = MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/environments/`,
+      body: TestStubs.Environments(),
+    });
+
+    render(<GroupEventDetailsContainer {...makeProps({organization})} />);
+    expect(await screen.findByTestId('loading-indicator')).toBeInTheDocument();
+
+    expect(environmentsCall).toHaveBeenCalledTimes(1);
```

Comment:
Can we add back the assert that it's finished loading as well?

Reply:
I thought about even just removing the L49 here and just assert that fetch is called